Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -260,6 +260,14 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+/** Set whether or not to log to stdout.
+ *
+ * @param enabled Whether or not stdout logging should be enabled.
+ *
+ * By default stdout logging is enabled.
+ */
+void kslog_setLogToStdout(bool enabled);
```

Comment:
Something like this? I've kept the current behavior of logging to stdout by default.

One related issue is that `kslog_setLogFilename` is called unconditionally from `kscrash_install`, so from a public API pov it is a bit confusing. You have to call it twice, before and after installing KSCrash, and you'll miss any logging that happens during the install (it will end up in ConsoleLog.txt). Perhaps I'm misunderstanding the use of `kslog_setLogFilename`?